client/src/pages/EventAdminReports/EventAdminReports.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import Spinner from 'components/Spinner';
+import {
+  InputGroup,
+  FormControl,
+  Container,
+  Row,
+  Col,
+  Button,
+  Modal,
+} from 'react-bootstrap';
+import { useQueryLookup } from 'hooks/admin';
+import { apiFetch } from 'utils/fetch';
+import Fuse from 'fuse.js';
+
+import ReportSearchResult from './ReportSearchResult';
+import ReportView from './ReportView';
+import ReportEditForm, { ReportEditFormValue } from './ReportEditForm';
+
+const reportSearchOptions = {
+  isCaseSensitive: true,
+  includeScore: true,
+  shouldSort: true,
+  includeMatches: true,
+  // findAllMatches: false,
+  location: 0,
+  threshold: 0.6,
+  distance: 100,
+  keys: [
+    'template',
+    'title',
+  ]
+}
+
+function EventAdminReports({ event, ...props }: EventAdminPageProps) {
+  const [reports, setReports] = React.useState<ApiReport[] | null>(null);
+  const [reportSearch, setReportSearch] = React.useState<Fuse<ApiReport> | null>(null);
+  const [searchQuery, setSearchQuery] = React.useState('');
+  const [show, setShow] = React.useState(false);
+  const queryLookup = useQueryLookup();
+
+  const [formValues, setFormValues] = React.useState<ReportEditFormValue>({
+    title: '',
+    template: '',
+  });
+
+  const getReports = async () => {
+    try {
+      const response = await apiFetch('/api/reports');
+      const json = await response.json();
+      const reportSearch = new Fuse<ApiReport>(json, reportSearchOptions);
+
+      setReports(json);
+      setReportSearch(reportSearch);
+    } catch (error) {
+      console.log('error', error);
```

Comment:
I guess this is a placeholder for to-be-implemented error UI, but using an Alert would create less risk of confusion due to invisible errors.

Reply:
I agree, we have a lot of console.log's left in catch clauses.  To be fixed eventually.